### PR TITLE
Some love for .mailmap

### PR DESCRIPTION
**Pull request by gacarrillor**
Thanks to a command from https://shane.io/2011/10/07/git-shortlog-and-mailmap.html it's easy to add multiple names. I just picked some of them, though.

Note that I'm making some (non-critical) decisions here. If you are being included in this PR and you prefer another name or e-mail (from the names/e-mails you've used in the past), just write a comment, so that we can make the adjustment.

For those that have no clue, you can run `git shortlog -nse`

@elpaso, @SrNetoChan, @roya0045, @AlisterH, @andreasneumann, @anitagraser, @arnaud-morvan, @AsgerPetersen, @borysiasty, @root676, @domi4484, @dmarteau, @signedav, @Gustry, @All4Gis, @giohappy, @gioman, @DelazJ, @suricactus, @Djedouas, @Jean-Roc, @jgrocha, @troopa81, @dakcarto, @luipir, @marioba, @neteler, @ghtmtt, @NEDJIMAbelgacem, @nicogodet, @nyalldawson, @olivierdalang, @dassau, @pblottiere, @PeterPetrik, @raymondnijssen, @haubourg, @rduivenvoorde, @saberraz, @pigreco, @Samweli, @uclaros, @ThomasG77, @tomchadwin, @tomasMizera, @tudorbarascu, @vcloarec, @kyngchaos, @yjacolin

### CODE REVIEW

**Comment by tomchadwin** [comment]
Thanks, @gacarrillor. I've barely contribued at all, but for the sake of accuracy, neither of those email addresses are mine. Happy to go with tomchadwin@astuntechnology.com, I guess. Thanks again.

File: .mailmap [hunk 1]
```diff
@@ -56,21 +137,83 @@ Nathan Woodrow <madmanwoo@gmail.com> <nathan_woodrow@technologyonecorp.com>
...
 
+Paul Blottiere <blottiere.paul@gmail.com> Blottiere Paul <blottiere.paul@gmail.com>
+
+Peter Petrik <zilolv@gmail.com> PeterPetrik <zilolv@gmail.com>
+
 Radim Blazek <radim.blazek@gmail.com> <rblazek@c8812cc2-4d05-0410-92ff-de0c093fc19c>
 
+Raymond Nijssen <r.nijssen@terglobo.nl> raymondnijssen <r.nijssen@terglobo.nl>
+
+Régis Haubourg <regis.haubourg@eau-adour-garonne.fr> Régis Haubourg <regis.haubourg@oslandia.com>
```
Comment (haubourg) [inline .mailmap:170]:
```suggestion
Régis Haubourg <regis@qgis.org> 
```

File: .mailmap [hunk 2]
```diff
@@ -42,10 +114,19 @@ Marco Hugentobler <marco.hugentobler@sourcepole.ch> <marco@sourcepole.ch>
 Marco Hugentobler <marco.hugentobler@sourcepole.ch> mhugent <marco.hugentobler@sourcepole.ch>
 Marco Hugentobler <marco.hugentobler@sourcepole.ch> mhugent <mhugent@c8812cc2-4d05-0410-92ff-de0c093fc19c>
 
+Mario Baranzini <mario@opengis.ch> <marioba@users.noreply.github.com>
+
+Markus Neteler <neteler@gmail.com> <neteler@osgeo.org>
+Markus Neteler <neteler@gmail.com> <neteler@oboe.localdomain>
```
[the remark this answers is not in the document]
Comment (gacarrillor) [reply .mailmap:120]: You've made at least one commit with `Markus Neteler <neteler@oboe.localdomain>`, that's why it should appear in the mapping, which will merge it to `Markus Neteler <neteler@gmail.com>`'s commits.

So, if you agree with your Gmail account being your preferred one, the mapping would stay as proposed in the PR.

File: .mailmap (hunk 1 above)
Comment (gacarrillor) [reply .mailmap]: The two e-mail accounts included in the mapping are found in the commit history. That's why they should appear in the file. 

They are now mapped to <regis@qgis.org> in 6c9a82c

**Comment by gacarrillor** [comment]
> Thanks, @gacarrillor. I've barely contribued at all, but for the sake of accuracy, neither of those email addresses are mine. Happy to go with [tomchadwin@astuntechnology.com](mailto:tomchadwin@astuntechnology.com), I guess. Thanks again.

Your previous commits are now mapped to your preferred e-mail in 6c9a82c :+1:

File: .mailmap [hunk 3]
```diff
@@ -1,20 +1,89 @@
...
 Giuseppe Sucameli <brush.tyler@gmail.com> <brushtyler@c8812cc2-4d05-0410-92ff-de0c093fc19c>
 
+Harrissou Sant-anna <delazj@gmail.com> DelazJ <delazj@gmail.com>
+
+Ivan Ivanov <ivan.ivanov@suricactus.com> <suricactus@users.noreply.github.com>
+
+Jacky Volpes <jacky.volpes@oslandia.com> <34267385+Djedouas@users.noreply.github.com>
+
+Jean-Roc Morreale <jr.morreale@enoreth.net> Jean Roc <jr.morreale@enoreth.net>
+Jean-Roc Morreale <jr.morreale@enoreth.net> MORREALE Jean-Roc <jr.morreale@enoreth.net>
```
Comment (Jean-Roc) [inline .mailmap:81]: hi, the email address is the same, you can merge _MORREALE Jean-Roc_ and _Jean Roc_ with _Jean-Roc Morreale_

Comment (gacarrillor) [reply .mailmap:81]: Right, the proposed mapping will merge all commits made by `Jean Roc <jr.morreale@enoreth.net>` and `MORREALE Jean-Roc <jr.morreale@enoreth.net>` into the preferred name/e-mail pair: `Jean-Roc Morreale <jr.morreale@enoreth.net>`.

**Comment by borysiasty** [comment]
Thanks a lot! I've added one more entry for my old SVN commits (using the github's online editor, so hopefully I didn't introduce yet another email to the pool :D)

File: .mailmap [hunk 4]
```diff
@@ -42,10 +115,19 @@ Marco Hugentobler <marco.hugentobler@sourcepole.ch> <marco@sourcepole.ch>
 Marco Hugentobler <marco.hugentobler@sourcepole.ch> mhugent <marco.hugentobler@sourcepole.ch>
 Marco Hugentobler <marco.hugentobler@sourcepole.ch> mhugent <mhugent@c8812cc2-4d05-0410-92ff-de0c093fc19c>
 
+Mario Baranzini <mario@opengis.ch> <marioba@users.noreply.github.com>
```
Comment (marioba) [inline .mailmap:118]:
```suggestion
Mario Baranzini <mario.baranzini@gmail.com>
```

Comment (gacarrillor) [reply .mailmap]: The two e-mail accounts included in the mapping are found in the commit history. That's why they should appear in the file.

They are now mapped to `Mario Baranzini <mario.baranzini@gmail.com>` in [679036d](https://github.com/gacarrillor/QGIS/commit/679036d7b27a00626c0acb8870674a1feb5bcda7)

**Comment by DelazJ** [comment]
And if ever you contributed to the docs repository, you might want to give a look to https://github.com/qgis/QGIS-Documentation/pull/8709

**Comment by gacarrillor** [comment]
Just for the record, all comments were already addressed.

**Review by troopa81 — APPROVED** [review]

**Comment by gacarrillor** [comment]
Thank you @troopa81 for the review, if there is something to do from my side, please let me know.

**Comment by troopa81** [comment]
> Thank you @troopa81 for the review, if there is something to do from my side, please let me know.

Nothing to do on your side, CI jobs were failing because of something not related to your PR (see https://github.com/qgis/QGIS/pull/55741), I restarted jobs once again, they should be green now :crossed_fingers: then I could merge

**Comment by troopa81** [comment]
And now mingw job is failing....

I already closed/reopened the issue 3 times to have a different CI issue each time for a file which has actually nothing to do with CI. 

@nyalldawson @elpaso @m-kuhn Could you please by-pass CI and merge this one ?

**Comment by gacarrillor** [comment]
Thank you @troopa81 and @elpaso for the review and merge! :+1:

**Pull request merged**